CodeGeneration/Sources/SyntaxSupport/DeclNodes.swift

```diff
@@ -467,6 +467,7 @@ public let DECL_NODES: [Node] = [
           .keyword(._resultDependsOnSelf),
           .keyword(.required),
           .keyword(.static),
+          .keyword(.transferring),
```

Comment:
Should this be underscored in addition to being behind an experimental flag? IIUC that’s the current best practice. But I might be wrong.

Reply:
@ahoppen it is an experimental flag that is only compiled into asserts builds, so it can't escape out into the wild. If I was compiling in the flag into non-asserts builds, I would agree.